Source/ModelGroups/Buffer_Misc.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using System.Numerics;
+using AssetGenerator.Runtime;
+using System.Collections.Generic;
+
+namespace AssetGenerator.ModelGroups
+{
+    internal class Buffer_Misc : ModelGroup
+    {
+        public override ModelGroupId Id => ModelGroupId.Buffer_Misc;
+
+        public Buffer_Misc(List<string> imageList)
+        {
+            var baseColorTexture = new Texture { Source = UseTexture(imageList, "BaseColor_Grey")};
+
+            // Track the common properties for use in the readme
+            CommonProperties.Add(new Property(PropertyName.BaseColorTexture, baseColorTexture.Source.ToReadmeString()));
+
+            Model CreateModel(Action<List<Property>, Runtime.MeshPrimitive> setProperties) 
+            {
+                var properties = new List<Property>();
+                Runtime.MeshPrimitive meshPrimitive = MeshPrimitive.CreateSinglePlane();
```

Comment:
```suggestion
                Runtime.MeshPrimitive meshPrimitive = MeshPrimitive.CreateSinglePlane(includeTextureCoords: false);
```

You can pass in false for the `includeTextureCoords` parameter to exclude texture coordinates.